src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,17 @@
+.stars {
+  display: flex;
+  justify-content: center;
+
+  &__star {
+    background-image: url(/src/images/star.svg);
+    width: $star-size;
+    height: $star-size;
+    background-repeat: no-repeat;
+    background-position: center;
+    margin-right: 4px;
+  }
+
+  &--4 :nth-child(-n + 4) {
+    background-image: url(/src/images/star-active.svg);
+  }
```

Comment:
It should be the same functionality as in the stars task, if the modifier is `stars--3` you should show 3 active stars and so on